src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -24,4 +41,61 @@
 @WebServlet("/list-organizations")
 public class ListOrganizationsServlet extends HttpServlet {
 
-}
\ No newline at end of file
+  /*
+   * This get request returns a list of organizations depending on its query parameters. 
+   * If no parameters are included, it will return a default list of organizations
+   */
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /* All get requests will return a maximum of 5 organization entities */
+    FetchOptions fetchOptions = FetchOptions.Builder.withLimit(5);
+    
+    /* TODO: Implement using pagination cursors- first implementation will just return 5 first entries */
+
+    /* This parameter is used to tell the servlet whether the user is requesting to see just the orgs they moderate*/
+    String displayUserOrgsParameter = request.getParameter("displayUserOrgs");
+    boolean displayUserOrgs = false;
+
+    /* If a parameter was sent & is set to 'true', then the displayUserOrgs boolean changes to true */
+    if ((displayUserOrgsParameter != null) && (displayUserOrgsParameter.equals("true"))) {
+      displayUserOrgs = true;
+    }
+
+    UserService userService = UserServiceFactory.getUserService();
+    boolean isUserLoggedIn = userService.isUserLoggedIn();
+
+    Query query;
+
+    if (isUserLoggedIn && displayUserOrgs) {
+      /* If the user is logged in and wants to just see their orgs, get their user ID & index with it*/
+      String userId = userService.getCurrentUser().getUserId();
+
+      query = new Query("Distributor").setFilter(new FilterPredicate("moderatorList",
```

Comment:
This logic generally looks good, but can you check the spacing? This looks odd in github reviews. Is there a formatter you can use?